sources/engine/Stride/Graphics/Image.cs

```diff
@@ -557,9 +557,9 @@ public static unsafe Image Load(byte[] buffer, bool loadAsSRGB = false)
         /// <remarks>This method support the following format: <c>dds, bmp, jpg, png, gif, tiff, wmp, tga</c>.</remarks>
         public static Image Load(Stream imageStream, bool loadAsSRGB = false)
         {
-            if (imageStream == null) throw new ArgumentNullException("imageStream");
+            if (imageStream == null) throw new ArgumentNullException("imageStream is null");
```

Comment:
I think adding "is null" is not necessary. You will receive following message : 
```bash
System.ArgumentNullException: Value cannot be null. (Parameter 'imageStream is null')
```

Reply:
will remove it.

Reply:
done